fe/fe-core/src/main/java/org/apache/doris/nereids/operators/AbstractOperator.java

```diff
@@ -33,4 +38,17 @@ public AbstractOperator(OperatorType type) {
     public OperatorType getType() {
         return type;
     }
+
+    public <R, C> R accept(PlanOperatorVisitor<R, C> visitor, PhysicalPlan<?, ?> physicalPlan, C context) {
+        return null;
+    }
+
+    public long getLimited() {
+        return limited;
+    }
+
+    public void setLimited(long limited) {
```

Comment:
Operator and Plan's properties should be immutable. So no setter.

Reply:
done